pyasic/miners/factory.py

```diff
@@ -620,6 +627,12 @@ async def _get_miner_web(self, ip: str) -> MinerTypes | None:
                     res = await self.send_web_command(ip, "/kaonsu/v1/brief", auth=auth)
                     if res is not None:
                         mtype = MinerTypes.MARATHON
+                if mtype == MinerTypes.HAMMER:
+                    res = await self.get_miner_model_hammer(ip)
+                    if "Hammer" in res:
```

Comment:
Maybe use res.upper() here, and make both the checks strict.  The else case can warn about an unknown miner subtype or something.